draft-ietf-tls-tls13.md

```diff
@@ -1383,10 +1383,7 @@ Structure of this message:
 
 %%% Key Exchange Messages
 
-       struct {
-           uint8 major;
-           uint8 minor;
-       } ProtocolVersion;
+       uint8 ProtocolVersionID[2];
```

Comment:
```
enum {
  ssl3_RESERVED (0x300),
  tls10 (0x301),
  tls11 (0x302),
  tls12 (0x303),
  tls13 (0x304),
  (65535)
} ProtocolVersion;
```

I don't see the point of this being `ProtocolVersionID` over just `ProtocolVersion`.